pandas/core/arrays/masked.py

```diff
@@ -851,6 +851,17 @@ def copy(self: BaseMaskedArrayT) -> BaseMaskedArrayT:
         mask = mask.copy()
         return type(self)(data, mask, copy=False)
 
+    def unique(self: BaseMaskedArrayT) -> BaseMaskedArrayT:
+        """
+        Compute the BaseMaskedArray of unique values.
+
+        Returns
+        -------
+        uniques : BaseMaskedArray
+        """
+        uniques, mask = algos._unique(self._data, self._mask)
```

Comment:
nitpick, can you rename the function to something non-private, e.g. unique_with_mask

Reply:
Sure, done